src/builders.rs

```diff
@@ -188,6 +191,38 @@ impl EditUserProfile {
 	}
 }
 
+impl EditRole {
+    /// Edit the role's name.
+    pub fn name(self, name: &str) -> Self{
+        EditRole(self.0.insert("name", name))
+    }
+
+    /// Bitwise of the enabled/disabled permissions.
+    pub fn permissions(self, perm: u64) -> Self{
+        EditRole(self.0.insert("permissions", perm))
+    }
+
+    /// Edit the sorting postion of the role.
+    pub fn position(self, pos: u64) -> Self{
+        EditRole(self.0.insert("position", pos))
+    }
```

Comment:
The position can't be modified on a PATCH for a single role; this needs to be done in a [batch guild roles PATCH](https://discordapp.com/developers/docs/resources/guild#modify-guild-role-positions), i.e. a separate `Discord::move_roles` method.